raiden_contracts/contracts/test/CustomToken.sol

```diff
@@ -84,4 +84,8 @@ contract CustomToken is StandardToken {
         owner_address.transfer(address(this).balance);
         assert(address(this).balance == 0);
     }
+
+    function decimals() public view returns (uint8 decimals) {
+        return _decimals;
+    }
```

Comment:
This should not be necessary. The getter is added by the compiler if `uint8 decimals` is `public`. But you can leave it.